src/main/scala/package.scala

```diff
@@ -19,6 +19,18 @@ package object suffuse {
   def eok()                             = 0
   def tryFuse(body: => Unit): Int       = Try(body) fold (_.toErrno, _ => eok)
 
+  // For example statsBy(path("/usr/bin").ls)(_.mediaType.subtype)
+  //
+  // 675   octet-stream
+  // 96    x-shellscript
+  // 89    x-perl
+  // 26    x-c
+  // ...
+  def statsBy[A, B](xs: Seq[A])(f: A => B): Unit = {
```

Comment:
Plus or minus. It's ridiculously handy and I get tired of needing it in every project.